app/controllers/api/v1/stories_controller.rb

```diff
@@ -49,12 +40,46 @@ def set_story
     # Only allow a list of trusted parameters through.
     def story_params
       params.require(:story).permit(
-        :subbject,
+        :subject,
         :hero,
         :place,
         :character,
         :object,
         :age
       )
     end
+
+    # In general I believe that the controller should not be responsible for
```

Comment:
Reading this, would you say that we should be writing services for these AI models (perhaps our first opportunity to leverage interactor composition as the workflows get complex). For example: hypothetically, in this case, a service that composes interactors to generate stories in `lib/chat-gpt/v3.5/turbo-instruct`?

Reply:
Yes the interactor library would be a good candidate. Do you think this should be part of MVP?

Reply:
Yes, the interactor library is a good candidate. Do you think it should be part of MVP?